test/train-sets/ref/help.stdout

```diff
@@ -429,6 +429,7 @@ Cost-sensitive Active Learning:
   --csa_debug                           print debug stuff for cs_active
 Cost Sensitive One Against All:
   --csoaa arg           One-against-all multiclass with <k> costs
+  --probabilities       predict probabilites of all classes
```

Comment:
does this predict probabilities or report probabilities?

Reply:
I believe it's doing doing both, but this is following the naming convention for oaa and csoaa_ldf.